src/components/KTableView/KTableView.vue

```diff
@@ -1167,9 +1175,10 @@ const setActualColumnWidths = (): void => {
   actualColumnWidths.value = widths
 }
 
-watch([columnVisibility, tableHeaders], (newVals) => {
+watch([columnVisibility, tableHeaders, hasExpandableRows], (newVals) => {
   const newVisibility = newVals[0]
   const newHeaders = newVals[1]
+  const newExpandableRows = newVals[2]
```

Comment:
referencing the new values by index, now that there are 3, seems less than ideal. Could you extract them as variables and then reference them below as needed?

```ts
const [newVisibility, newHeaders, newExpandableRows] = newVals
```

Reply:
Done